views/posts/index.hbs

```diff
@@ -1,49 +1,10 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-  <script type="text/javascript" src="/javascripts/comment.js"> </script>
```

Comment:
I think this script tag needs to be somewhere in the html

Reply:
just added it to the layout